OpenSim/Actuators/TorqueActuator.cpp

```diff
@@ -53,7 +53,7 @@ TorqueActuator::TorqueActuator()
 }
 //_____________________________________________________________________________
 // Constructor with given body names.
-TorqueActuator::TorqueActuator(const Body& bodyA, const Body& bodyB,
+TorqueActuator::TorqueActuator(const PhysicalFrame& bodyA, const PhysicalFrame& bodyB,
```

Comment:
>(const PhysicalFrame& bodyA, const PhysicalFrame& bodyB,

Should probably be renamed to const PhysicalFrame& frameA, and frameB.  They may not be bodies, but instead frames fixed to bodies.